pkg/csi_driver/multishare_controller.go

```diff
@@ -303,7 +315,7 @@ func (m *MultishareController) startAndWaitForInstanceDeleteOrShrink(ctx context
 	}
```

Comment:
startAndWaitForInstanceDeleteOrShrink() calls some ops manager functions which need the StatusError() handling

Reply:
I don't think any functions in the ops manager need the StatusError handling, if we wrap errors appropriately, then we can just call StatusError in multishare_controller only.

Reply:
Ack. This is cleaner to handle it only in controller